modules/pubmaticServerBidAdapter.js

```diff
@@ -170,10 +155,12 @@ export const spec = {
   */
   isBidRequestValid: bid => {
     if (bid && bid.params) {
-      return mandatoryParamCheck('publisherId', bid.params.publisherId) &&
-        mandatoryParamCheck('adUnitId', bid.params.adUnitId) &&
-        mandatoryParamCheck('divId', bid.params.divId) &&
-        mandatoryParamCheck('adUnitIndex', bid.params.adUnitIndex);
+      const requiredParamsArr = ['publisherId', 'adUnitId', 'profId'];
```

Comment:
why divid is optional now ?
it is required as KGP might be _DIV_, we had discussed this